yes, the only failed check was `urnai/sc2/actions/collectables.py:1:1: I001 [*] Import block is un-sorted or un-formatted`. I didn't manage to fix it manually and the `--fix` flag doesn't work. We had already discussed this and another solution was proposed a while back, but i tried that and it didn't work either

In summary, i can't solve these import issues.

--- a/experiments/solves/solve_collectables_sb3.py
+++ b/experiments/solves/solve_collectables_sb3.py
@@ -5,20 +5,20 @@ import numpy as np
 
 sys.path.append(os.path.abspath(os.path.join(os.path.dirname(__file__), '..', '..')))
 
+import wandb
 from absl import app
 from gymnasium import spaces
 from pysc2.env import sc2_env
 from stable_baselines3 import PPO
 from stable_baselines3.common.monitor import Monitor
+from wandb.integration.sb3 import WandbCallback
 
-import wandb
 from urnai.environments.stablebaselines3.custom_env import CustomEnv
 from urnai.sc2.actions.collectables import CollectablesActionSpace
 from urnai.sc2.environments.sc2environment import SC2Env
 from urnai.sc2.rewards.collectables import CollectablesReward
 from urnai.sc2.states.collectables import CollectablesMethod, CollectablesState
 from urnai.trainers.stablebaselines3_trainer import SB3Trainer
-from wandb.integration.sb3 import WandbCallback
 
 
 def declare_wandb_run(config_dict : dict, run_id : str = None):

--- a/urnai/sc2/actions/collectables.py
+++ b/urnai/sc2/actions/collectables.py
@@ -6,6 +6,7 @@ from urnai.actions.action_space_base import ActionSpaceBase
 from urnai.sc2.actions import sc2_actions_aux as scaux
 from urnai.sc2.actions.sc2_actions import raw_functions_classes as sc2_actions
 
+
 class CollectablesActionSpace(ActionSpaceBase):
 
     def __init__(self):

